src/utils/facilities/formatCostCenterFromServer.spec.js

```diff
@@ -0,0 +1,29 @@
+import omit from 'lodash.omit';
+import formatCostCenterFromServer from './formatCostCenterFromServer';
+
+describe('utils/facilities/formatCostCenterFromServer', function() {
+  let expectedCostCenter;
+  let formattedCostCenter;
+  let costCenter;
+
+  beforeEach(function() {
+    costCenter = fixture.build('costCenter', null, {
+      fromServer: true
+    });
+    expectedCostCenter = omit(
+      {
+        ...costCenter,
+        createdAt: costCenter.created_at,
+        organizationId: costCenter.organization_id,
+        updatedAt: costCenter.updated_at
+      },
+      ['created_at', 'organization_id', 'updated_at']
+    );
+
+    formattedCostCenter = formatCostCenterFromServer(costCenter);
+  });
+
+  it('converts the object keys to camelCase', function() {
+    expect(formattedCostCenter).to.deep.equal(expectedCostCenter);
+  });
+});
```

Comment:
There should be a test about formatting children properties (like `facilities`). I think there is an example in: https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/utils/facilities/formatFacilityFromServer.spec.js

Reply:
https://github.com/ndustrialio/contxt-sdk-js/pull/34/files#diff-31f28e295fbcb24bd72f174ad08447c0R39

Is that what you're looking for?

Reply:
Yes 👍